mlflow/store/tracking/dbmodels/models.py

```diff
@@ -100,7 +100,8 @@ def to_mlflow_entity(self):
         """
         Convert DB model to corresponding MLflow entity.
 
-        :return: :py:class:`mlflow.entities.Experiment`.
+        Returns:
+            Description of the return value.
```

Comment:
This seems not correct? Also could we keep :py:class: for all return values in this file?

Reply:
Great catch!